### DISCUSSION TITLE
CVEs

### DISCUSSION THREAD

**Discussion by Seirdy** [Alex313031/thorium#94] [General]
Thorium doesn't seem to push emergency fixes for CVEs, which is understandable given the resources required to re-build. I think this should be explicitly clarified in the project README, so people understand the risks before making it their default browser.

**Comment by Alex313031** [comment]
@Seirdy That was a good idea, and unfortunately I didn't see this until now. However, Thorium is now being built to track the stable Chromium/Chrome releases, and so will have whatever fixes official releases have, usually within a few days of upstream release (I need to time to rebase, fix bugs, and build.

**Reply by Alex313031** [reply]
@Seirdy Added a SECURITY.md file.
